tests/functional/adapter/test_aliases.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+from dbt.tests.adapter.aliases.test_aliases import BaseAliases, BaseSameAliasDifferentSchemas
+
+_MACROS__BIGQUERY_CAST_SQL = """
```

Comment:
@ChenyuLInx @gshank would writing a macro like this in the adapter fully replace what was in the core version of the test? tests are failing becuase the `test_expect_value` is undefined which it tries to say is a `deps` issue because we are calling on a macro that doesn't exist. have tried as it is now and just reapplying the `cast.sql `macro directly into the adapter version to same result.